network-api/networkapi/wagtailpages/pagemodels/customblocks/base_fields.py

```diff
@@ -9,12 +10,7 @@
 
 base_fields = [
     ('paragraph', blocks.RichTextBlock(
-        features=[
-            'bold', 'italic', 'large',
-            'h2', 'h3', 'h4', 'h5',
-            'ol', 'ul',
-            'link', 'hr',
-        ],
+        features=base_rich_text_options + ['large', 'h2', 'h3', 'h4', 'h5', 'ol', 'ul', 'hr'],
```

Comment:
Looking at

```
full_content_rich_text_options = base_rich_text_options + ['h2', 'h3', 'h4', 'h5', 'ol', 'ul', 'hr']
```

this looks like it should be `features=full_content_rich_text_options  + ['large']`?